examples/evaluators/golang/simple/main.go

```diff
@@ -26,41 +26,52 @@ package main
 
 import (
 	"context"
-	"fmt"
-	"log"
-	"os"
+	"flag"
 	"strings"
 	"time"
 
 	"github.com/GoogleCloudPlatform/open-match/config"
 	om_messages "github.com/GoogleCloudPlatform/open-match/internal/pb"
+	"github.com/GoogleCloudPlatform/open-match/internal/set"
 	redishelpers "github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis"
+	"github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis/ignorelist"
 	"github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis/redispb"
 	"github.com/gobs/pretty"
 	"github.com/gomodule/redigo/redis"
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 )
 
+var (
+	lgr    *log.Logger
+	cfg    *viper.Viper
+	err    error
+	pool   *redis.Pool
+	dryrun bool
+)
+
 func main() {
+	flag.BoolVar(&dryrun, "dryrun", false, "Print eval results, but do not take approval or rejection actions")
+	flag.Parse()
 
 	//Init Logger
-	lgr := log.New(os.Stdout, "MMFEval: ", log.LstdFlags)
-	lgr.Println("Initializing example MMF proposal evaluator")
+	lgr = log.New()
+	lgr.Info("Initializing example MMF proposal evaluator")
 
 	// Read config
+	// This golang example uses the Open Match config module.  If you are
+	// writing an evaluator in another language, most libraries/modules that
+	// can parse a YAML file will work.
 	lgr.Println("Initializing config...")
-	cfg, err := config.Read()
+	cfg, err = config.Read()
 	if err != nil {
 		panic(nil)
 	}
 
 	// Connect to redis
-	pool, err := redishelpers.ConnectionPool(cfg)
-	if err != nil {
-		lgr.Fatal(err)
-	}
-	defer pool.Close()
-
+	// As per https://www.iana.org/assignments/uri-schemes/prov/redis
+	// redis://user:secret@localhost:6379/0?foo=bar&qux=baz // redis pool docs: https://godoc.org/github.com/gomodule/redigo/redis#Pool
+	pool := redishelpers.ConnectionPool(cfg)
```

Comment:
Please revert this back. This method returns pool, err now. This line does not concern itself with the credentials so it's unclear why this comment is necessary. If anything the comment should live in redishelpers.